pkg/commands/options/publish.go

```diff
@@ -98,31 +106,43 @@ func AddPublishArg(cmd *cobra.Command, po *PublishOptions) {
 		"Whether to just use KO_DOCKER_REPO without additional context (may not work properly with --tags).")
 }
 
-func packageWithMD5(base, importpath string) string {
-	hasher := md5.New() //nolint: gosec // No strong cryptography needed.
+func pathJoin(paths ...string) string {
```

Comment:
This seems more complicated than just using `path.Join` -- is there a reason for it?

Reply:
That's a left over from removed option. On it.